Makefile

```diff
@@ -132,6 +132,14 @@ docker-buildx-telemetry-proxy: ## Build and push docker image for the manager fo
 	docker buildx rm project-v3-builder && \
 	rm Dockerfile.cross )
 
+.PHONY: republish-kube-rbac-proxy
+republish-kube-rbac-proxy: ## Republish kube-rbac-proxy image
+	( git clone git@github.com:brancz/kube-rbac-proxy.git ~/kube-rbac-proxy && \
+	cat kube-rbac-proxy/publish.sh > ~/kube-rbac-proxy/scripts/publish.sh && \
```

Comment:
what's the point of this line?

Reply:
I needed to change several lines in the original publish script (of the kube-rbac-proxy repo) to make it publish our version to our repo. and I thought it will be more clear to change the original script like this and override it.